app/config/assets/css/odk-survey.css

```diff
@@ -341,7 +341,32 @@ Eventually we should make a jqm theme for this.
 
 .instanceRowControl{
   float: right;
-}
+  display: flex;
+  margin-right: auto;
+  gap: 5px;
+}
+
+.instanceRowControl .openInstance {
+   border-top-left-radius: 8px!important;
```

Comment:
Hello - great work! 👍 
One small question regarding the CSS; Is the '!important' tag necessary? 
It is really a kind of last resort in CSS (i.e. when something "more specific" exists in the css inheritance hierarchy).. However - if it is indeed needed here, then just ignore this comment! 😄